### DISCUSSION TITLE
Guthub app storage location

### DISCUSSION THREAD

**Discussion by jperrucci1963** [community/community#40249] [Mobile]
Can I move GitHub app to an external storage location?

**Comment by ossanna16** [comment]
Hi there @jperrucci1963 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by RubenJ27** [comment]
The answer is yes, but if you connect your git to that external device, it will be bound to the computer where you logged in when you connected the external device, so it is recommended to install it on the storage where the OS is hosted.
